tools/collate_catchment_attributes.py

```diff
@@ -70,16 +85,19 @@ def out_file_dest(aggregate_df,outFile):
     parser.add_argument('-s', '--sierra-test-input', help='layer containing sierra test by hydroId',required=True)
     parser.add_argument('-o','--output-csv-destination',help='location and name for output csv',required=True)
     parser.add_argument('-l','--link-elev-table',help='elev table for linking sierra tests to hydrotable on locationid and hydroid',required=True)
-    
+    parser.add_argument('-lc','--lulc', help='data set with lulc pixel counts', required=True)
+
     args = vars(parser.parse_args())
 
     fim_directory = args['fim_directory']
     sierra_test_input = args['sierra_test_input']
     output_csv_destination = args['output_csv_destination']
     link_elev_table = args['link_elev_table']
+    pixel_dir = args['lulc']
```

Comment:
Recommend keeping the variable names consistent with argument names. I also recommend a more descriptive variable name, e.g. `nlcd_pixel_count_dir`